spec/game.spec.js

```diff
@@ -0,0 +1,89 @@
+var TowerOfHanoi 
+
+beforeEach(function () {
+  TowerOfHanoi = new GameBoard();
+})
+
+describe("Game can move pegs correctly", () => {
+  it("Is an object", () => {
+    expect(typeof TowerOfHanoi).toEqual("object")
+  })
+
+  it("Has correct initial game board", () => {
+    expect(TowerOfHanoi.board).toEqual([[5,4,3,2,1],[],[]])
+  })
+
+  it("Can move disks correctly", () => {
+    expect(TowerOfHanoi.move(0,1)).toEqual([[5,4,3,2],[1],[]])
+  })
+
+  it("Doesn't move disk if it moves onto a smaller disk than itself", () => {
+    TowerOfHanoi.move(0,1);
+    expect(TowerOfHanoi.move(0,1)).toEqual("You cannot move a larger disc on top of a smaller one")
+  })
+
+  it("Doesn't move disk if the origin peg is empty", () => {
+    expect(TowerOfHanoi.move(1,2)).toEqual("Cannot move disk from an empty peg")
+  })
+})
+
+describe("Game formats game board correctly", () => {
+
+  it("Creates formatted string of game board from game board", () => {
+    expect(TowerOfHanoi.stringFormattedGameBoard()).toEqual(['--- 5 4 3 2 1', '---', '---'])
+  })
+
+  it("Creates and array of h2 elements from game board", () => {
+    expect(TowerOfHanoi.createPegElementsArray()[0].nodeName).toEqual("DIV")
+  })
+
+  it("Creates/resets board correctly", () => {
+    TowerOfHanoi.move(0,1)
+    TowerOfHanoi.move(0,2)
+    expect(TowerOfHanoi.resetBoard()).toEqual([[5,4,3,2,1],[],[]])
```

Comment:
Not a big one but you can create a const in a const file or in this file with your defaultBoardMock and use it instead of repeating yourself.